lib/entry-state.js

```diff
@@ -1,8 +1,9 @@
 const { getEntry } = require('./../services/contentful')
+const { isObject, isArray } = require('lodash')
 
 async function getPublishedEntry (entry) {
   try {
-    return await getEntry(entry.sys.id)
+    return await getEntry(entry.sys.id, entry.sys.contentType.sys.id)
```

Comment:
For what do we need the content type here?

Reply:
we are performing a query and in that case the `content_type` is required